### DISCUSSION TITLE
ubuntu 설치 과정 오류

### DISCUSSION THREAD

**Discussion by leejaehun1** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#105] [2024년도 기타 질문]
늦게 문의드려 죄송합니다.
혼자 해결해보려했으나 되지가 않아 문의남깁니다.
영상에 맞게 usb통해서 바이오스 우선순위를 바꾸어 실행했습니다.
usb를 제거하면 부트로더가 아닌 윈도우로 바로 실행되어 곤혹입니다.
vmware사용해서 과제해야할까요

**Comment by HyeongKeunHong** [comment]
USB를 제거 한 후 바이오스 우선순위 설정을 해보셨을까요?
